src/microbe_stage/OrganismStatisticsPanel.cs

```diff
@@ -0,0 +1,613 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using Godot;
+
+/// <summary>
+///   Displays organism statistics calculated by an editor component
+/// </summary>
+public partial class OrganismStatisticsPanel : PanelContainer
+{
+    [Export]
+    public bool ShowHealthStat;
+
+    [Export]
+    public bool ShowSizeStat;
+
+    [Export]
+    public bool ShowStorageStat;
+
+    [Export]
+    public bool ShowSpeedStat;
+
+    [Export]
+    public bool ShowRotationSpeedStat;
+
+    [Export]
+    public bool ShowDigestionSpeedStat;
+
+    [Export]
+    public bool ShowDigestionEfficiencyStat;
+
+#pragma warning disable CA2213
+
+    [Export]
+    public LabelSettings ATPBalanceNormalText = null!;
+
+    [Export]
+    public LabelSettings ATPBalanceNotEnoughText = null!;
+
+#pragma warning restore CA2213
+
+    private readonly StringBuilder atpToolTipTextBuilder = new();
+
+#pragma warning disable CA2213
+
+    [Export]
+    private CellStatsIndicator sizeLabel = null!;
+
+    [Export]
+    private CellStatsIndicator speedLabel = null!;
+
+    [Export]
+    private CellStatsIndicator rotationSpeedLabel = null!;
+
+    [Export]
+    private CellStatsIndicator hpLabel = null!;
+
+    [Export]
+    private CellStatsIndicator storageLabel = null!;
+
+    [Export]
+    private CellStatsIndicator digestionSpeedLabel = null!;
+
+    [Export]
+    private CellStatsIndicator digestionEfficiencyLabel = null!;
+
+    [Export]
+    private Control basicStatsSeparator = null!;
+
+    [Export]
+    private Control movementStatsSeparator = null!;
+
+    [Export]
+    private Control digestionStatsSeparator = null!;
+
+    [Export]
+    private Label generationLabel = null!;
+
+    [Export]
+    private Control atpBalancePanel = null!;
+
+    [Export]
+    private Label atpBalanceLabel = null!;
+
+    [Export]
+    private Label atpProductionLabel = null!;
+
+    [Export]
+    private Label atpConsumptionLabel = null!;
+
+    [Export]
+    private SegmentedBar atpProductionBar = null!;
+
+    [Export]
+    private SegmentedBar atpConsumptionBar = null!;
+
+    [Export]
+    private CompoundBalanceDisplay compoundBalance = null!;
+
+    [Export]
+    private CompoundStorageStatistics compoundStorageLastingTimes = null!;
+
+    [Export]
+    private CustomRichTextLabel notEnoughStorageWarning = null!;
+
+    [Export]
+    private CheckBox calculateBalancesAsIfDay = null!;
+
+    [Export]
+    private CheckBox calculateBalancesWhenMoving = null!;
+
+    [Export]
+    private Button processListButton = null!;
+
+    [Export]
+    private ProcessList processList = null!;
+
+    [Export]
+    private CustomWindow processListWindow = null!;
+
+    [Export]
+    private LightConfigurationPanel lightConfigurationPanel = null!;
+
+#pragma warning restore CA2213
+
+    private LightLevelOption selectedLightLevelOption =
+        LightLevelOption.Current;
```

Comment:
```suggestion
    private LightLevelOption selectedLightLevelOption = LightLevelOption.Current;
```
Wouldn't this fit on a single line?